src/main/java/com/programmers/jpaboard/domain/BaseEntity.java

```diff
@@ -0,0 +1,21 @@
+package com.programmers.jpaboard.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+public abstract class BaseEntity {
+
+    @CreatedDate
+    @Column(updatable = false)
+    protected LocalDateTime createdAt;
+
```

Comment:
createdBy도 앞으로 업데이트 될 일이 없을 것이라 생각이 되어지는데 어떻게 생각하시나요?!

```suggestion
@Column(updateable = false)
```

Reply:
반영 : [0c508fd](https://github.com/prgrms-be-devcourse/springboot-board-jpa/pull/235/commits/0c508fd75c785293c30102333c6e6d65b70c4a16)
- 네 맞습니다! 너무 좋네요. 바로 바꿨습니다.